packages/in_app_purchase/lib/src/billing_client_wrappers/purchase_wrapper.dart

```diff
@@ -74,46 +75,55 @@ class PurchaseWrapper {
 
   /// The unique ID for this purchase. Corresponds to the Google Payments order
   /// ID.
+  @JsonKey(defaultValue: '')
   final String orderId;
 
   /// The package name the purchase was made from.
+  @JsonKey(defaultValue: '')
   final String packageName;
 
   /// When the purchase was made, as an epoch timestamp.
+  @JsonKey(defaultValue: 0)
   final int purchaseTime;
 
   /// A unique ID for a given [SkuDetailsWrapper], user, and purchase.
+  @JsonKey(defaultValue: '')
   final String purchaseToken;
 
   /// Signature of purchase data, signed with the developer's private key. Uses
   /// RSASSA-PKCS1-v1_5.
+  @JsonKey(defaultValue: '')
   final String signature;
 
   /// The product ID of this purchase.
+  @JsonKey(defaultValue: '')
   final String sku;
 
   /// True for subscriptions that renew automatically. Does not apply to
   /// [SkuType.inapp] products.
   ///
   /// For [SkuType.subs] this means that the subscription is canceled when it is
   /// false.
-  final bool isAutoRenewing;
+  final bool? isAutoRenewing;
```

Comment:
What would a null value mean here? If this is intentional, it should be documented.